ci-operator/step-registry/ipi/conf/ibmcloud/kms-key/ipi-conf-ibmcloud-kms-key-ref.yaml

```diff
@@ -5,6 +5,16 @@ ref:
     name: "4.12"
     tag: upi-installer
   commands: ipi-conf-ibmcloud-kms-key-commands.sh
+  env:
+  - name: IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY
+    default: "true"
+    documentation: Using a Key Protect key for volume encryption of controlplane machines.
+  - name: IBMCLOUD_COMPUTE_ENCRYPTION_KEY
+    default: "true"
```

Comment:
Suggest the default value is `false` for master and computer to save KMS quota in this step, if you need to cover then together, you can enable it in job level.